From ac40f7dec47df9c48fa55d90be67ea4cbebcb09d Mon Sep 17 00:00:00 2001
From: Kai Uwe Broulik <kde@privat.broulik.de>
Date: Wed, 11 Oct 2017 16:03:46 +0200
Subject: Revert "Remove pointless data copy in lockscreen delegate"

The switch user greeter accesses it to get the "vtNumber" when you click the "Switch" button.

CHANGELOG: Fixed being unable to switch users from the Switch User screen

This reverts commit cc8647481095f5631ef8136945ee5d2d06981fd6.
---
 lookandfeel/contents/components/UserDelegate.qml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lookandfeel/contents/components/UserDelegate.qml b/lookandfeel/contents/components/UserDelegate.qml
index b74e657..acd08e3 100644
--- a/lookandfeel/contents/components/UserDelegate.qml
+++ b/lookandfeel/contents/components/UserDelegate.qml
@@ -28,6 +28,7 @@ Item {
 
     property bool isCurrent: true
 
+    readonly property var m: model
     property string name
     property string userName
     property string avatarPath
--
cgit v0.11.2
